genesis-tools/genesis-populate/src/state_dump.rs

```diff
@@ -44,7 +44,7 @@ impl StateDump {
             let mut roots_files = dir;
             roots_files.push(GENESIS_ROOTS_FILE);
             let mut file = File::create(roots_files)?;
-            let data = self.roots.try_to_vec()?;
+            let data = borsh::to_vec(&self.roots)?;
             file.write_all(&data)?;
```

Comment:
[I think we can directly write to the file here since borsh supports to_writer](https://docs.rs/borsh/latest/borsh/fn.to_writer.html)